misaligned.spec.js

```diff
@@ -0,0 +1,45 @@
+const {expect} = require('chai');
+const  print_color_map = require('./misaligned.js');
+// Import the print_color_map function
+// Redirect console.log to a custom function
+const originalLog = console.log;
+const capturedLogs = [];
+
+console.log = function (...args) {
+    capturedLogs.push(args.join(' '));
+    originalLog.apply(console, args);
+};
+
+//console.log(capturedLogs);
```

Comment:
Don't leave commented code in commits. Remove before committing.

Reply:
My bad. Forgot to remove before committing. I have removed in